tools/sof-tplgreader.py

```diff
@@ -46,6 +59,25 @@ def loadFile(self, filename, sofcard=0):
                 pipeline_dict['fmt'] = pipeline_dict['fmts'].split(' ')[0]
                 pipeline_dict['rate_min'], pipeline_dict['rate_max'] = self._key2str(cap, 'rate')
                 pipeline_dict['ch_min'], pipeline_dict['ch_max'] = self._key2str(cap, 'channels')
+                if pcm_type == "both":
+                    pipeline_dict["type"] = "capture"
+                    # copy pipeline and change values
+                    pb_pipeline_dict = copy.copy(pipeline_dict)
+                    pb_pipeline_dict["type"] = "playback"
+                    cap = pcm["caps"][pcm['playback']]
+                    pga = pgas[pcm['capture']]
```

Comment:
Different value between 'playback' and 'capture' ?

Reply:
yes, PGA and EQ is total the different value for playback and capture